project/SparkRedshiftBuild.scala

```diff
@@ -52,6 +52,7 @@ object SparkRedshiftBuild extends Build {
         "Spark 1.5.0 RC2 Staging" at "https://repository.apache.org/content/repositories/orgapachespark-1141",
       libraryDependencies ++= Seq(
         "com.amazonaws" % "aws-java-sdk-core" % "1.9.40" % "provided",
+        "com.amazonaws" % "aws-java-sdk-s3" % "1.9.40" % "provided",
```

Comment:
We've discussed this a little bit more offline and have decided to switch back to marking these dependencies as `provided`, but with an additional comment that captures some of this discussion and a re-wording of the relevant line in the README. I'll submit a PR for this today.